.github/workflows/publish.yml

```diff
@@ -1,30 +1,20 @@
-name: Publish
+name: Publish from comments
 
 on:
-  release:
-    types: [ published ]
-  workflow_dispatch:
+  issue_comment:
+    types: [created]
 
 jobs:
   publish:
-    strategy:
-      matrix:
-        package: [
-          "dio",
-          "plugins/cookie_manager",
-          "plugins/http2_adapter",
-          "plugins/native_dio_adapter"
-        ]
-    defaults:
-      run:
-        working-directory: dio
+    # https://github.com/cfug/dio/issues/1633
+    if: github.event.issue.number == 1633
```

Comment:
Do you consider that if you push a tag, a new version be published. Rather than commenting in issue.

Reply:
Generally, this will simplify the process because tags can be pushed multiple times by mistake operations. Also, using the issue comment could provide a better trace of who started the publishment.

Reply:
Okay. So how about pushing a tag after version publishing?

Reply:
> Okay. So how about pushing a tag after version publishing?

It'll be handled by the action, see https://github.com/CaiJingLong/dio/releases/tag/dio_v5.0.1-dev.2.